### ISSUE TITLE
About the loss in pytorch

### ISSUE THREAD

**Issue by Yidoudoujiang** [ialhashim/DenseDepth#99]
Thanks for your code! But why the number of the loss in pytorch is two not three? Did you forget the grad loss?

**Comment by atemysemicolon** [comment]
Please refer to issue #83 
Pytorch support is experimental - left to us to design the loss function.
